esp8266-fastled-webserver/include/simplehacks/timestamp.h

```diff
@@ -0,0 +1,126 @@
+/**
```

Comment:
SIMPLE FILE REVIEW

Allows the date and time of last change to a file to be added as `constexpr` compatible value (integers, strings).

Will be used in /product JSON endpoint.  May use to define `last-modified` header for webserver.